Components/Grid/grid.html

```diff
@@ -76,8 +76,126 @@
             <div class="content-container">
                 <h1>Grid</h1>
                 <p>
-                    Grids will come here.
+                    Grids are used to build responsive layouts in grid style fashion (Columns and Rows).
                 </p>
+                <br>
+                <br>
+                <br>
+                <br>
+                <h2>Grid of 2 Items with 50-50 ratio</h2>
+                <div class="grid-container">
```

Comment:
Why are you using two divs instead of apending a class on grid1
like for example "grid-container grid-1"

Reply:
Thanks for suggestion, your point makes sense, I will work on it in next PR.